Submodels/MTFLO_call.py

```
@@ -97,30 +105,23 @@ def GenerateProcess(self,
-        self.process = subprocess.Popen([self.fpath, self.analysis_name], 
-                                        stdin=subprocess.PIPE, 
-                                        stdout=subprocess.PIPE, 
-                                        stderr=subprocess.PIPE,
-                                        shell=True, 
+        self.process = subprocess.Popen([self.process_path, self.analysis_name],
+                                        stdin=subprocess.PIPE,
+                                        stdout=subprocess.PIPE,
+                                        stderr=subprocess.DEVNULL,
                                         text=True,
                                         bufsize=1,
                                         )
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)